### ISSUE TITLE
fread fails to read 1-column file

### ISSUE THREAD

**Issue by st-pasha** [Rdatatable/data.table#2322]
Here's the file:
```
require(data.table)
fread("default payment next month
0.5524199604988098
0.24839693307876587
0.11578908562660217
0.12682904303073883
0.11796239763498306
0.14982038736343384
0.05930440500378609
0.16200894117355347
0.19789715111255646
0.12607525289058685
0.2425299435853958
0.07361350208520889
0.11026645451784134
0.5387383699417114")
```
produces
```
Error in fread("...") : 
  Line 1 has too few fields when detecting types. Use fill=TRUE to pad with NA. 
Expecting 4 fields but found 1: <<      0.5524199604988098>>
```
CC @arnocandel

**Comment by MichaelChirico** [comment]
don't you just need to specify sep?

**Comment by st-pasha** [comment]
I can, but I shouldn't have to

**Comment by st-pasha** [comment]
This is probably related to #1073

**Comment by MichaelChirico** [comment]
Closed by #2451
